src/main/java/lavalink/client/io/LavalinkRestClient.java

```diff
@@ -0,0 +1,168 @@
+package lavalink.client.io;
+
+import com.sedmelluq.discord.lavaplayer.player.AudioLoadResultHandler;
+import com.sedmelluq.discord.lavaplayer.tools.FriendlyException;
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack;
+import com.sedmelluq.discord.lavaplayer.track.BasicAudioPlaylist;
+import com.sedmelluq.lava.common.tools.DaemonThreadFactory;
+import lavalink.client.LavalinkUtil;
+import lavalink.client.io.jda.JdaLink;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.function.Consumer;
+
+public final class LavalinkRestClient {
+
+    private static final ExecutorService EXECUTOR_SERVICE = Executors.newCachedThreadPool(new DaemonThreadFactory("Lavalink-RestExecutor"));
+    private static final Consumer<IOException> DEFAULT_ERROR_CONSUMER = Throwable::printStackTrace;
```

Comment:
Don't print to sdout. Use SLF4J to log exceptions.

Also consider removing this. It is redundant when you can just define the lambda inline.